crates/zed/src/main.rs

```diff
@@ -191,12 +191,25 @@ fn main() {
         watch_file_types(fs.clone(), cx);
 
         languages.set_theme(cx.theme().clone());
+        cx.windows().iter_mut().for_each(|&mut w| {
+            let bg = cx.theme().window_background();
+            w.update(cx, |_, wcx| {
```

Comment:
We should just use `cx` here, not `wcx`:

```suggestion
            w.update(cx, |_, cx| {
```